sql/schema.sql

```diff
@@ -10,7 +10,7 @@ CREATE TABLE IF NOT EXISTS repos
 , owner      TEXT      NOT NULL       
 , name       TEXT      NOT NULL
 , descr      TEXT      NOT NULL
-, categories TEXT      ARRAY
+, categories TEXT      ARRAY NOT NULL
```

Comment:
I think that `ARRAY` should be closer to type rather than to constraint. So shit should be something like:

```sql
categories TEXT ARRAY                 NOT NULL
```